src/apiCalls.js

```diff
@@ -0,0 +1,9 @@
+export const getBooks = () => {
```

Comment:
Great job here with adding a little error handling. We should consider displaying an error to the user incase we encounter an error. I will add an issue for this.